grommet/grommet
# Add Dutch translations for messages and icons

Signed-off-by: Werner Donné <werner.donne@pincette.biz>



#### What does this PR do?
It adds two localization files for Dutch, one for the messages and another for the icons.
#### Where should the reviewer start?
In /grommet/src/js/messages
#### What testing has been done on this PR?
None
#### How should this be manually tested?
Set the browser to Dutch and see if the affected localized components have changed.
#### Any background context you want to provide?
No.
#### What are the relevant issues?
https://github.com/grommet/grommet/issues/1223 (closed)
#### Screenshots (if appropriate)
None
#### Do the grommet docs need to be updated?
No.
#### Should this PR be mentioned in the release notes?
Not really.
#### Is this change backwards compatible or is it a breaking change?
It is compatible.
